text/0000-reserve-element.md

```diff
@@ -0,0 +1,55 @@
+- Start Date: 2015-12-03
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes we reserve the keyword `element` for all objects in a Refract
+tree.
+
+# Motivation
+
+This allows us to not worry about domain-specific knowledge when parsing a
+Refract tree. Right now, we have to rely on something out of band to tell us
+when something MAY be refracted, but if we rely on the reserved keyword
+`element`, we can know if an object is a normal object or a Refract Object.
+
+Note: we are already doing this in Minim.
+
+# Design
+
+This RFC proposes to reserve `element` in the entire tree. This means that
+anywhere the `element` keyword is found, a parser SHOULD treat the object as a
```

Comment:
anywhere the `element` key is present, right? Calling it keyword sounds weird and allows ambiguity of it being present in a value of an object.